airflow/providers/openlineage/plugins/facets.py

```diff
@@ -33,7 +33,7 @@ class AirflowMappedTaskRunFacet(BaseFacet):
     @classmethod
     def from_task_instance(cls, task_instance):
         task = task_instance.task
-        from airflow.providers.openlineage.utils import get_operator_class
+        from airflow.providers.openlineage.utils.utils import get_operator_class
```

Comment:
utils.utils 🤣

Reply:
Any suggestion how to improve that? :)

Reply:
I don't think we should change that now or at all.